dockerfiles/keycloak/kc_realm_user.sh

```diff
@@ -18,7 +18,7 @@ if [ "${CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD}" == "false" ]; then
 fi
 
 cat /scripts/che-realm.json.erb | \
-                                sed -e "s@<%= scope\.lookupvar('che::che_server_url') %>@${PROTOCOL}://${CHE_HOST}@" \
+				sed -e "s@<%= scope\.lookupvar('che::che_server_url') %>@${PROTOCOL}://che-${NAMESPACE}.${ROUTING_SUFFIX}@" \
```

Comment:
Please make sure that Che on kubernetes won't be broken after reverting changes introduced in https://github.com/eclipse/che/pull/10509/files#diff-48382692058266aaf66d0fc1105bb4bbR21